src/Engine/GameVersion.java

```diff
@@ -12,7 +12,7 @@
   int majorRev, minorRev, hotfix;
   public GameVersion()
   {
-    majorRev = 4; // For substantial, save-breaking changes to the game logic.
+    majorRev = 5; // For substantial, save-breaking changes to the game logic.
```

Comment:
Not save protection, or control modifications, or AI restructuring, or enforced air unit crashes, or any of the other numerous recent (tm) changes. Nope. We draw the line at colorful overlays.

Reply:
I was originally going to forget and then probably do it on the test branch.
I don't really care about the `GameVersion` when it's not in the hands of users, lol.

Save protection starts out null, so it might not technically break the save, but that's a valid major rev.
AI restructuring doesn't actually affect the save file, so it'd be a minor rev (until we start saving AI state).
Air unit crashes doesn't change the save file either. Might even be "hotfix" tier.

Reply:
Hah, fair enough. I was being facetious. Certainly we needed to roll it before the next real release, so it's the accumulation of all those things that matters.